src/url-builder.coffee

```diff
@@ -45,6 +45,9 @@ module.exports =
     if config.ssl
       params.push 'ssl=true'
 
+    if config.authDatabase
+      params.push 'authSource=' + config.authDatabase
```

Comment:
NIT: would be a bit nicer with string interpolation: `"authSource=#{config.authDatabase}"`